src/evaluation/WorkspaceManager.jl

```diff
@@ -22,37 +26,38 @@ Base.@kwdef mutable struct Workspace
     original_ACTIVE_PROJECT::Union{Nothing,String}=nothing
 end
 
+const SN = Tuple{ServerSession, Notebook}
+
 "These expressions get evaluated whenever a new `Workspace` process is created."
-process_preamble() = quote
```

Comment:
This needs to be a function because we call project_relative_path, which uses RelocatableFolders.jl, see #1852

Reply:
I missed the interpolation. I'll update.